app/experimenter/nimbus-ui/src/components/FormOverview/index.test.tsx

```diff
@@ -82,111 +75,107 @@ describe("FormOverview", () => {
     const expected = {
       name: "Foo bar baz",
       hypothesis: "Some thing",
-      application: "firefox-desktop",
+      application: "DESKTOP",
     };
 
     const onSubmit = jest.fn();
     render(<Subject {...{ onSubmit }} />);
 
-    await act(async () => {
-      const submitButton = screen.getByText("Create experiment");
-      await fireEvent.click(submitButton);
-      expect(onSubmit).not.toHaveBeenCalled();
-    });
-
-    await act(async () => {
-      await fillOutNewForm(expected);
-    });
+    const submitButton = screen.getByText("Create experiment");
+    await act(async () => void fireEvent.click(submitButton));
+    expect(onSubmit).not.toHaveBeenCalled();
 
-    await act(async () => {
-      const submitButton = screen.getByText("Create experiment");
-      await fireEvent.click(submitButton);
-    });
+    await act(async () => fillOutNewForm(expected));
+    await act(async () => void fireEvent.click(submitButton));
 
     expect(onSubmit).toHaveBeenCalled();
     expect(onSubmit.mock.calls[0][0]).toEqual(expected);
   });
 
-  it("with missing public description, does not allow submit", async () => {
-    const { data } = mockExperimentQuery("boo", { publicDescription: "" });
+  it("with existing experiment data, asserts field values before allowing submit and next", async () => {
```

Comment:
Ohh, you know I think we actually want to allow saving pieces of an experiment without filling out required fields. I think a user should be able to leave "Public Description" blank if they wanted to only update say, the hypothesis, and save that, to update the public description later.

However, we could double check this with Ana and maybe do this in a follow up as I'm not _entirely_ certain of this.

Reply:
Updated to make optional